wolfcrypt/src/signature.c

```diff
@@ -175,7 +175,15 @@ int wc_SignatureVerify(
                 int is_valid_sig = 0;
 
                 /* Perform verification of signature using provided ECC key */
-                ret = wc_ecc_verify_hash(sig, sig_len, hash_data, hash_len, &is_valid_sig, (ecc_key*)key);
+                do {
+                #ifdef WOLFSSL_ASYNC_CRYPT
+                    ret = wc_AsyncWait(ret, &((ecc_key*)key)->asyncDev,
```

Comment:
Asymmetric op is blocking but hash isn't.
I was expecting wc_SignatureVerify() and wc_SignatureGenerate() to be asynchronous.
Are asynchronous versions going to be added?